src/CubicInterpolation/Interpolant.hpp

```diff
@@ -41,13 +41,14 @@ auto back_transform_prime(T1 trafo, T2 const &axis, double f, T3 df, T3 x) {
 }
 
 double _find_parameter(std::function<double(double)> const &f,
-                       std::function<double(double)> const &df, double x,
-                       Axis const &axis);
+                       std::function<double(double)> const &df, Axis const &axis,
+                       double initial_guess);
 
-template <typename T1> auto find_parameter(T1 const &inter, double val, double x) {
+template <typename T1>
+auto find_parameter(T1 const &inter, double val, double initial_guess) {
```

Comment:
Would it be possible (and useful) to have a default value for initial_guess = nan, so that the user can pass no argument (instead of a nan)?

Reply:
I think it would be a reasonable default value. 
The problem is, that `nan` is defined in the `<cmath>` lib. 
Including it in the header file will increase the build time significant if the interpolant is included often.